src/cloudmask.jl

```diff
@@ -61,25 +18,34 @@ Convert a 3-channel false color reflectance image to a 1-channel binary matrix;
 
 """
 function create_cloudmask(
-    false_color_image::Union{Matrix{RGB{Float64}}, Matrix{RGBA{N0f8}}};
-    prelim_threshold::Float64=Float64(110 / 255),
-    band_7_threshold::Float64=Float64(200 / 255),
-    band_2_threshold::Float64=Float64(190 / 255),
-    ratio_lower::Float64=0.0,
-    ratio_upper::Float64=0.75,
-)::BitMatrix
-    mask_cloud_ice, clouds_view = _get_masks(
-        false_color_image,
-        prelim_threshold=prelim_threshold,
-        band_7_threshold=band_7_threshold,
-        band_2_threshold=band_2_threshold,
-        ratio_lower=ratio_lower,
-        ratio_upper=ratio_upper,
-    )
+    modis_721::AbstractArray{T};
+    prelim_threshold::Real=(110 / 255),
+    band_7_threshold::Real=(200 / 255),
+    band_2_threshold::Real=(190 / 255),
+    ratio_lower::Real=0.0,
+    ratio_upper::Real=0.75,
+)::BitMatrix where {T<:Union{AbstractRGB,TransparentRGB}}
+    modis_band02 = green.(modis_721)
+    modis_band07 = red.(modis_721)
```

Comment:
I like that this approach makes it specific that this particular algorithm requires the MODIS 7-2-1 image type, and uses the green and red functions to get the channel info.